style.css

```diff
@@ -4,68 +4,55 @@ body {
   padding: 0;
 }
 
-.header {
-  height: 48px;
-  background: #3c3a39;
-  display: flex;
-  justify-content: space-around;
-  align-items: center;
-  padding-bottom: 24px;
-  margin-top: 0;
-}
-
-.hamburger {
-  padding-top: 58px;
-  margin-right: 24px;
-}
-
-.hide-desktop {
-  display: inline;
-}
-
-.hide-mobile {
-  display: inline;
+.social-media img {
+  margin-right: 16px;
 }
 
-.main-container {
-  background-image: url("image/illustration-header-mobile.png");
+.social-media {
+  padding: 18px;
+  margin-top: 22px;
+  display: flex;
+  height: 400px;
   background-repeat: no-repeat;
-  background-position: right bottom;
-  background-position-y: 200px;
-  padding-bottom: 0;
-  height: 580px;
-  background-color: #1c1a19;
+  margin-right: 0;
+  list-style: none;
 }
 
 .container {
   padding-left: 24px;
   padding-right: 24px;
-  color: #fff;
-  font-weight: bold;
-  font-family: "Roboto", sans-serif;
-  margin-bottom: -55px;
 }
 
-.navbar {
+.header {
+  height: 48px;
+  background: #3c3a39;
   display: flex;
-  flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  width: 100%;
+  padding-bottom: 24px;
+  margin-top: 0;
 }
 
-.navbar a {
-  float: left;
-}
+/* .header a {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: white;
+  text-decoration: none;
+  padding-top: 58px;
+} */
```

Comment:
- [ ] [REQUIRED]  Please remove any line of code that you are not using as a good practice of making your code clean.😊